packages/frontend-v2/hooks/useStudentWithPlans.ts

```diff
@@ -10,8 +11,8 @@ type StudentResponse = Omit<
 
 type UseStudentReturn = StudentResponse & {
   isLoading: boolean;
-  mutateStudent: KeyedMutator<GetStudentResponse>;
-  student?: GetStudentResponse;
+  mutateStudent: KeyedMutator<StudentModel<string>>;
```

Comment:
Is this changed because we need the id type of string for each course when updating the plan?

Reply:
yeaaa, it's cause we don't return the student but rather a processed students that has their plans populated with dnd ids